atomic_client.py

```diff
@@ -77,6 +77,10 @@ def images(self):
     def vulnerable(self):
         return self.dbus_object.VulnerableInfo(dbus_interface="org.atomic", timeout = 2147400)
 
+    @polkit.enable_proxy
+    def ps(self):
+        return self.dbus_object.Ps(dbus_interface="org.atomic", timeout = 2147400)
```

Comment:
possibly this is the timeout though set for the test python client however. in cockpit, the client doesn't timeout. stefw had recommended this timeout length for me.